### ISSUE TITLE
Statetest support for ConstatinopleFix

### ISSUE THREAD

**Issue by holiman** [paritytech/parity-ethereum#10260]
I just tried restarting the fuzzing framework on the ConstatinopleFix / Petersburg / StPetersfork ruleset. The fuzzer relies on generating statetests, however, running it showed that Parity does not yet support the new statetest chain `ConstantinopleFix`

Error msg:
```
Unable to load the test file: unknown variant `ConstantinopleFix`, expected one of `EIP150`, `EIP158`, `Frontier`, `Homestead`, `Byzantium`, `Constantinople`, `EIP158ToByzantiumAt5`, `FrontierToHomesteadAt5`, `HomesteadToDaoAt5`, `HomesteadToEIP150At5` at line 1 column 450

```

**Comment by holiman** [comment]
Hey folks, any progress here? I'd consider this pretty important...

**Comment by 5chdn** [comment]
Sorry, yes, this is in progress.

**Comment by tomusdrw** [comment]
Closed via #10323
